hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2573,14 +2580,16 @@
 
             - Provision an RMA_UNLOCK token in OTP.
             - Reset the chip.
+            - Ensure device id and manufacturing state can be read through the LC JTAG.
```

Comment:
@cindychip 
I unmapped this test for now so it will show up as something we need to "finish".
Let me know if this makes sense to you.

I realize now I was probably the person who recommended it be done from C-side...sorry!

Reply:
Sounds good! No worries at all.